README.md

```diff
@@ -1,6 +1,133 @@
-# website
+# Website
+
 The website for the Civic Tech Index
 
+To develop the site, you'll need to first clone the repository on to your computer. For new Git users, see the [Using Git](#using-git) section below. <br><br>
+
+# OVERVIEW
+**Set up**
+1. [Join the Repo Team](#step-1-become-a-member-of-the-repository-team)
+
+2. [Using Git](#using-git) and [Fork the Repo](#step-2-fork-the-repository)
+
+3. [Clone to your local machine](#step-3-clone-your-online-repository-to-your-local-computer)
+
+4. [Set up Docker](#step-4-setting-up-docker)
+
+**Before you start working on an issue**
+
+5. [Read CTI's Site Architecture to get acquainted with how the website is structured](https://github.com/CTI-website-frontend/website/wiki/CTI-Site-Architecture)
```

Comment:
I made up this wiki page to match theirs but we don't have one like this. Should we build one like this? Or just remove this reference and renumber the list? @ExperimentsInHonesty ?

Reply:
Resolved - removed this line